enterprise/coderd/prebuilds/reconcile.go

```diff
@@ -310,6 +310,15 @@ func (c *StoreReconciler) ReconcilePreset(ctx context.Context, ps prebuilds.Pres
 		return nil
 	}
 
+	// Nothing has to be done.
+	if !ps.Preset.UsingActiveVersion && actions.IsNoop() {
```

Comment:
@dannykopping Do we need `!ps.Preset.UsingActiveVersion`? I think we can skip reconciliation for all presets if `actions.IsNoop() == true`?

Reply:
I guess, yeah, but if the preset is part of an inactive version then we shouldn't be logging anything for that; for active presets which have no operations to perform, that's at least diagnostically useful.

Consider an installation that's been using prebuilds for a while and they have >100 inactive preset versions; that'll log a tonne for no value.

Reply:
okay, let's keep it as it is

Reply:
I mean here we log with debug level which is probably okay?
```go
// Nothing has to be done.
if !ps.Preset.UsingActiveVersion && actions.IsNoop() {
	logger.Debug(ctx, "skipping reconciliation for preset - nothing has to be done",
		slog.F("template_id", ps.Preset.TemplateID.String()), slog.F("template_name", ps.Preset.TemplateName),
		slog.F("template_version_id", ps.Preset.TemplateVersionID.String()), slog.F("template_version_name", ps.Preset.TemplateVersionName),
		slog.F("preset_id", ps.Preset.ID.String()), slog.F("preset_name", ps.Preset.Name))
	return nil
}
```

and then exit early without additional logs with info/warn log level

Reply:
Nope we can keep the debug log 👍